extensions/blocks/testimonial/edit.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { RichText, MediaPlaceholder, BlockControls, AlignmentToolbar } from '@wordpress/editor';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from '../../utils/i18n';
+
+export default class TestimonialEdit extends Component {
+	onChangeName = value => void this.props.setAttributes( { name: value } );
+	onChangeTitle = value => void this.props.setAttributes( { title: value } );
+	onChangeContent = value => void this.props.setAttributes( { content: value } );
+	onChangeAlign = value => void this.props.setAttributes( { align: value } );
+	onChangeMedia = media =>
+		void this.props.setAttributes( { mediaUrl: media.url, mediaId: media.id } );
+
+	render() {
+		const { attributes, className, isSelected } = this.props;
+		const hasMedia = attributes.mediaUrl && attributes.mediaId;
+
+		const alignmentClass = attributes.align && `is-aligned-${ attributes.align }`;
+
+		return (
+			<Fragment>
+				<BlockControls>
+					<AlignmentToolbar value={ attributes.align } onChange={ this.onChangeAlign } />
```

Comment:
Do you need this or can you just set `align` in the `supports` definition?

Reply:
I've tried `supports: { align: [ … ] }` but it automatically applies some styles to the block for left/right which made it float around other blocks (like an image block would). I need the block to stay where it is and only use the alignment attr to restyle things inside. I couldn't get it with out of box alignment so I made my own implementation. Would be great to hear from somebody else whether it makes sense.

Reply:
`supports.align` is block aligment and for content alignment you'd need something custom as you've done here. I think even core Paragraph is custom. Maybe some day core will have `supports.contentAlignment` or similar :-)

https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-registration/#supports-optional